### PR TITLE
Route

### PR DESCRIPTION
**Pull request by amaaisha**
Basic Route completed.

### CODE REVIEW

File: src/Components/Profile/EditProfile.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const handleLogin = (props) => {
+  auth.login(() => {
+    props.history.push("/logIn")
+  });
+};
+const EditProfile = () => {
+    return (
+      <div>
+        <h2>Edit Profile</h2>
+        onClick = {handleLogin}
```
Comment (stanjhae) [inline src/Components/Profile/EditProfile.js]: What is happening here?

File: src/Components/Profile/ViewProfile.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const handleLogin = (props) => {
+  auth.login(() => {
+    props.history.push("/logIn")
+  });
+};
+const ViewProfile = () => {
+    return (
+      <div>
+        <h2>View Profile</h2>
+        onClick = {handleLogin}
```
Comment (stanjhae) [inline src/Components/Profile/ViewProfile.js]: Same comment as editProfile

File: src/Components/Profile/EditProfile.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import auth from "../Routes/protectedRoutes/auth";
+
+
+const handleLogin = (props) => {
+  auth.login(() => {
```
Comment (stanjhae) [inline src/Components/Profile/EditProfile.js]: This should be in a componentDidMount or a useEffect.

File: src/Components/Routes/protectedRoutes/auth.js [hunk 4]
```diff
@@ -5,12 +5,12 @@ class Auth {
 
   login(callback) {
     this.authenticated = true;
-    callback();
+    setTimeout(callback, 100);
```
Comment (stanjhae) [inline src/Components/Routes/protectedRoutes/auth.js]: why do you have a time out here?

File: src/Components/Routes/protectedRoutes/auth.js [hunk 5]
```diff
@@ -5,12 +5,12 @@ class Auth {
...
   }
 
   logout(callback) {
     this.authenticated = false;
-    callback();
+    setTimeout(callback, 100);
```
Comment (stanjhae) [inline src/Components/Routes/protectedRoutes/auth.js]: same as above

File: src/Components/Routes/protectedRoutes/protectedRoute.js [hunk 6]
```diff
@@ -13,7 +13,7 @@ export const ProtectedRoute = ({component: Component, ...rest}) => {
           return (
             <Redirect
               to={{
-                pathname: "/",
+                pathname: "/logIn",
```
Comment (stanjhae) [inline src/Components/Routes/protectedRoutes/protectedRoute.js]: please change to `log-in`

File: src/Components/Profile/EditProfile.js (hunk 3 above)
Comment (amaaisha) [reply src/Components/Profile/EditProfile.js]: uhhh gosh does it really matter?

File: src/Components/Profile/EditProfile.js (hunk 1 above)
Comment (amaaisha) [reply src/Components/Profile/EditProfile.js]: something 😆

File: src/Components/Routes/protectedRoutes/auth.js (hunk 4 above)
Comment (amaaisha) [reply src/Components/Routes/protectedRoutes/auth.js]: sorry oo

**Pull request merged**